package.json

```diff
@@ -74,7 +74,8 @@
     "sinon": "19.0.2",
     "sinon-chai": "3.7.0",
     "uglify-js": "3.19.3",
-    "wait-on": "7.2.0"
+    "wait-on": "7.2.0",
+    "yarn-audit-fix": "^10.0.9"
```

Comment:
Do we need this going forward or was it just for the audit run?

Reply:
I just left it in as IMO anything to speed up/help with vuln patches is worth having, especially if we're doing it regularly. But I'm happy to add/remove it locally each time instead if required?

Reply:
Ok yeah think that justifies the cost of one more dependency, leave it in 👍